test/conftest.py

```diff
@@ -1,6 +1,8 @@
 import shutil
 
 from pathlib import Path
+from packaging.version import Version
```

Comment:
@simaishi Can you check and see if this is available as an RPM from the standard repos?

Reply:
🤞🏻 it's also used by `ansible-core` so my working assumption was that we had solved it for the necessary dependencies.

Reply:
ansible-core has it bundled, but we do have an access to the standalone rpm as well.